src/handwritten/stream_readers.py

```diff
@@ -218,9 +208,11 @@ def read_many_sample(
 
         self._verify_array(data, number_of_samples_per_channel, False, True)
 
-        return _read_analog_f_64(
-            self._handle, data, number_of_samples_per_channel,
-            timeout)
+        _, samps_per_chan_read = self._interpreter.read_analog_f64(
+            self._handle, number_of_samples_per_channel,
+            timeout, FillMode.GROUP_BY_CHANNEL.value, data)
```

Comment:
What PR is updating the interpreter functions to take data arrays? Your branch and the target branch doesn't have a data input after the `FillMode`

Reply:
Yes. This is quite confusing but the function call arguments change is present in `_libary_interpreter.py` of another PR https://github.com/ni/nidaqmx-python/pull/295.